### PR TITLE
Remove use of package keyword

### PR DESCRIPTION
**Pull request by ben-kaufman**

### CODE REVIEW

File: contracts/schemes/UpgradeScheme.sol [hunk 1]
```diff
@@ -27,7 +27,8 @@ contract UpgradeScheme is VotingMachineCallbacks, ProposalExecuteInterface, Init
 
     event ProposalExecuted(
         address indexed _avatar,
-        bytes32 indexed _proposalId
+        bytes32 indexed _proposalId,
+        int256 _param
```
Comment (orenyodfat) [inline contracts/schemes/UpgradeScheme.sol]: why is this needed ?
should it be a boolean ?

Comment (ben-kaufman) [reply contracts/schemes/UpgradeScheme.sol]: It makes things slightly easier in the subgraph and we have it in all other contracts like that.

Comment (orenyodfat) [reply contracts/schemes/UpgradeScheme.sol]: should it be a boolean ?
should it be called _decision ?

Comment (ben-kaufman) [reply contracts/schemes/UpgradeScheme.sol]: Not sure, it's an int everywhere else, I think the voting machine returns int so that's why.

Comment (orenyodfat) [reply contracts/schemes/UpgradeScheme.sol]: this are due to history reasons , we now have the chance to rewrite history :) 
I think it should be boolean , name `decision` .
does subgraph look for this event ? why ?

Comment (ben-kaufman) [reply contracts/schemes/UpgradeScheme.sol]: to mark the proposal as executed and in this case also update the contract info versions.

Comment (ben-kaufman) [reply contracts/schemes/UpgradeScheme.sol]: changed
